<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PassengerInfoPage</class>
 <widget class="QWidget" name="PassengerInfoPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>911</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>219</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    
    border-radius: 5px;    /* 设置边框圆角为50像素，以创建椭圆形状 */
 
    font-size: 20px;       /* 设置字体大小 */
}
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditIdNumber">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>290</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    
    border-radius: 5px;    /* 设置边框圆角为50像素，以创建椭圆形状 */
 
    font-size: 20px;       /* 设置字体大小 */
}
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPhoneNumber">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>360</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    
    border-radius: 5px;    /* 设置边框圆角为50像素，以创建椭圆形状 */
 
    font-size: 20px;       /* 设置字体大小 */
}
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSubmit">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>480</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 24px;
    color: white;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                  stop:0 #FDC245, stop:1 #FB9E23);
border-radius: 10px;

}
QPushButton:hover {
background-color:#FAC85C;
}</string>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>100</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    background-color: skyblue; /* 背景颜色设置为蓝色 */
    color: white;          /* 文字颜色设置为白色 */
	padding: 1px;         /* 设置内边距 */
    font-size: 32px;       /* 设置字体大小 */
	border: 1px solid skyblue;
 border-radius: 10px;    /* 设置边框圆角为50像素，以创建椭圆形状 */
}



</string>
   </property>
   <property name="text">
    <string>添加乘机人</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>220</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: cadetblue; /* 背景颜色设置为蓝色 */
    color: white;          /* 文字颜色设置为白色 */
    border-radius: 5px;    /* 设置边框圆角为50像素，以创建椭圆形状 */
    padding: 2px;         /* 设置内边距 */
    font-size: 24px;       /* 设置字体大小 */
}
</string>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>290</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: cadetblue; /* 背景颜色设置为蓝色 */
    color: white;          /* 文字颜色设置为白色 */
    border-radius: 5px;    /* 设置边框圆角为50像素，以创建椭圆形状 */
    padding: 2px;         /* 设置内边距 */
    font-size: 24px;       /* 设置字体大小 */
}
</string>
   </property>
   <property name="text">
    <string>身份证</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>360</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: cadetblue; /* 背景颜色设置为蓝色 */
    color: white;          /* 文字颜色设置为白色 */
    border-radius: 5px;    /* 设置边框圆角为50像素，以创建椭圆形状 */
    padding: 2px;         /* 设置内边距 */
    font-size: 24px;       /* 设置字体大小 */
}
</string>
   </property>
   <property name="text">
    <string>手机号</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
